code/modules/games/cards/lotto.dm

```diff
@@ -46,12 +46,15 @@
 
 /obj/item/toy/lotto_ticket/attackby(obj/item/weapon/S, mob/user)
 	if(!revealed)
+		if(!user.is_holding_item(src))
+			to_chat(user, "<span class='notice'>The film is too tough to scratch without holding the ticket in your hand!</span>")
+			return 1
```

Comment:
Does this mean you can't scratch the things on a table or something? Please remove it if so, that scrambling isn't fun.

Reply:
No, if I allow people to scratch things on a table then their hands won't get ashed if using the Supermatter ticket.

Reply:
Then add a special case for the SM ticket, don't mess the other tickets' rhythm. Regular tickets need nothing special but the SM one needs you to hold it.
"Strange forces stop you from scratching the film. Get a strong grip on it and try again." or similar.